### DISCUSSION TITLE
ödev yükleme

### DISCUSSION THREAD

**Discussion by eceenur** [gusanmaz/bmb212-algorithms-2024#9] [Q&A]
Merhaba hocam ödev yüklemeyi yapamadım anlayamadım

**Comment by gusanmaz** [comment]
Merhaba Ecenur,

README dosyasinda gerekli detaylar var. Tam olarak nerede bir sorun yasiyorsunuz?

Github'u kod commit etmeye yabanciysaniz asagidaki linkteki aciklamalari takip edebilirsiniz.

https://chatgpt.com/share/f2843be4-5607-4026-99f9-73f72c20db3d
